test/Microsoft_Tests/src/SQLServer_Spec.enso

```diff
@@ -8,12 +8,15 @@ from Standard.Table.Errors import Invalid_Column_Names, Inexact_Type_Coercion, D
 
 import Standard.Database.DB_Column.DB_Column
 import Standard.Database.DB_Table.DB_Table
+import Standard.Database.Feature.Feature
```

Comment:
Could do this via autoscoping - instead of importing this just use `..` before constructor.